src/hotspot/share/compiler/compilerDirectives.hpp

```diff
@@ -34,11 +34,11 @@
 
   //      Directives flag name,    type, default value, compile command name
   #define compilerdirectives_common_flags(cflags) \
-    cflags(Enable,                  bool, false, X) \
-    cflags(Exclude,                 bool, false, X) \
-    cflags(BreakAtExecute,          bool, false, X) \
-    cflags(BreakAtCompile,          bool, false, X) \
-    cflags(Log,                     bool, LogCompilation, X) \
+    cflags(Enable,                  bool, false, Enable) \
```

Comment:
This flag only has a meaning when working with directives files. Doesn't it just add confusion to have a default from CompileCommand?

Reply:
agree. take it off.